test/helpers/utils.js

```diff
@@ -26,3 +27,10 @@ exports.checkExistentFiles = (files, directory) => checkFiles('file', files, dir
 
 exports.getFileContent = filePath =>
   fs.readFileSync(exports.getTestDirectory(filePath), { encoding: 'utf-8' });
+
+exports.getCommands = () => {
+  require('dotenv').config();
+  const enviroment = process.env.DEPLOYED;
+  const commandsLint = enviroment === '0' ? [...commandLinter, ...commands] : commands;
```

Comment:
It's a little bit tricky use a environment variable for this, i don't see why is necessary

Reply:
it is necessary, to be able to know which dependencies of the slinter will be installed according to the CI

Reply:
DEPLOYED still has a tricky name, check if [this default variables](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) (TRAVIS=true for example could be one of them) for travis do the trick for you

Reply:
ok!